pkg/keyspace/keyspace.go

```diff
@@ -233,6 +234,12 @@ func (manager *Manager) CreateKeyspace(request *CreateKeyspaceRequest) (*keyspac
 			request.Config[UserKindKey] = config[UserKindKey]
 		}
 	}
+	// Set default value of GCManagementType to KeyspaceLevelGC for NextGen
+	if kerneltype.IsNextGen() {
+		if _, ok := request.Config[GCManagementType]; !ok {
```

Comment:
Maybe it's better to do this: 
```suggestion
		if v, ok := request.Config[GCManagementType]; !ok || len(v) == 0 {
```

Reply:
BTW maybe it's better to comment that default keyspace is not affected as it's created during bootstrap